src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -389,4 +389,23 @@
     </StringProperty.DataSource>
   </StringProperty>
 
+  <BoolProperty Name="RunRoslynAnalyzersDuringBuild"
```

Comment:
For easier future review, can we keep the properties in this file in alphabetical order please.

Reply:
"Roslyn" was deliberately not used in any API name as per original API review. I would consider calling this `RunAnalyzersDuringBuild` or just `RunAnalyzers`.

Reply:
> "Roslyn" was deliberately not used in any API name as per original API review

Yes, that is true. However, the analyzers have become popular and well known as `Roslyn analyzers`, so much so that even [docs pages](https://docs.microsoft.com/en-us/visualstudio/code-quality/code-analysis-for-managed-code-overview?view=vs-2019) uses the term `Roslyn analyzers` almost everywhere. During the UX review almost everyone unanimously felt that we should use `Roslyn analyzers` as that is the term that developer community relates to much more then any other term.

Reply:
Based on the offline discussions, we have decided to not use "Roslyn" in the property pages as well as the documentation. Fixed with https://github.com/dotnet/project-system/pull/5094/commits/5173428a2f3a0d48a376664a90153061bb5f7b90